src/faebryk/core/graphinterface.py

```diff
@@ -123,7 +124,8 @@ def connect(self, other: Self, linkcls=None) -> Self:
         self.G.add_edge(self, other, link=link)
 
         if logger.isEnabledFor(logging.DEBUG):
-            logger.debug(f"GIF connection: {link}")
+            with exceptions_to_log():
```

Comment:
I had to add this because with logging at debug it failed to import [this file](https://github.com/atopile/faebryk/blob/593fd39ec5674ad7fd0fadf68744acf4c91f59d4/src/faebryk/library/TI_CD4011BE.py#L13)

The stringification of some edge in the graph was hitting a `NotNone` assertion

This is the best solution IMO though because we shouldn't be killing the program with debug exceptions that might just be caused by import-time execution

Reply:
Thanks for clarifying!